soh/soh/SohMenuBar.cpp

```diff
@@ -853,8 +853,18 @@ void DrawEnhancementsMenu() {
         if (ImGui::BeginMenu("Graphics"))
         {
             if (UIWidgets::PaddedEnhancementCheckbox("Use Alternate Assets", "gAltAssets", true, false)) {
+                if (CVarGetInteger("gAltAssets", 0) == 0) {
+                    CVarSetInteger("gAltLinkEquip", 0);
+                    CVarSetInteger("gMMBottles", 0);
+                }
                 ShouldClearTextureCacheAtEndOfFrame = true;
             }
+            if (CVarGetInteger("gAltAssets", 0) == 1) {
+                UIWidgets::PaddedEnhancementCheckbox("Alternate Equipment Loading", "gAltLinkEquip", true, false);
+                UIWidgets::Tooltip("Use alternate loading method for loading Link's assets");
+                UIWidgets::PaddedEnhancementCheckbox("Majora's Mask Bottles", "gMMBottles", true, false);
+                UIWidgets::Tooltip("Use alternate loading method Bottles, mimicing Majora's Mask");
```

Comment:
To somebody who has no idea what these options do, these tooltips don't quite say enough about what will happen when you turn them on compared to what's happening when they're off.  For the first one, you've already got a great description at the top of this PR; just condense it here to something like "Display Link's equipment separately from his body, allowing Timeless equipment to display properly on child Link and equipment to display properly on custom Link models."  

Similarly, for the bottles, you could say "Display bottle contents separately from the bottle itself, similar to Majora's Mask."

Reply:
Will do. I'm also gonna take a lot of the function renaming here and pull it back to zret itself due to how much i've sifted through figuring this out.